### PR TITLE
Set-CsUser - Param updates for TPM 3+

### PR DESCRIPTION
**Pull request by pbafna03**

### CODE REVIEW

File: skype/skype-ps/skype/Set-CsUser.md [hunk 1]
```diff
@@ -98,6 +98,8 @@ Indicates whether the user is allowed to make audio/visual (A/V) calls by using
 If set to True, the user will largely be restricted to sending and receiving instant messages.
 
 You cannot disable A/V communications if a user is currently enabled for remote call control, Enterprise Voice, and/or Internet Protocol private branch exchange (IP-PBX) soft phone routing.
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:102]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 2]
```diff
@@ -121,6 +123,8 @@ If you disable an account by using the Enabled parameter, the information associ
 If you later re-enable the account by using the Enabled parameter, the associated account information will be restored.
 This differs from using the `Disable-CsUser` cmdlet to disable a user account.
 When you run the `Disable-CsUser` cmdlet, all the Skype for Business Server data associated with that account is deleted.
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:127]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 3]
```diff
@@ -222,6 +226,8 @@ Accept wildcard characters: False
 The URI of the remote call control telephone gateway assigned to the user.
 The LineServerUri is the gateway URI, prefaced by "sip:".
 For example: sip:rccgateway@litwareinc.com
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:230]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 4]
```diff
@@ -245,6 +251,8 @@ Private lines are often used for personal phone calls or for business calls that
 
 The private line value should be specified using the E.164 format, and be prefixed by the "TEL:" prefix.
 For example: TEL:+14255551297.
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:255]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 5]
```diff
@@ -269,6 +277,8 @@ To make and receive phone calls over the Internet, the user must be enabled for
 For details, see the parameter EnterpriseVoiceEnabled.
 
 To be enabled for remote call control, a user must have both a LineUri and a LineServerUri.
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:281]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 6]
```diff
@@ -287,6 +297,8 @@ Accept wildcard characters: False
 ### -SipAddress
 Unique identifier (similar to an email address) that allows the user to communicate using SIP devices such as Skype for Business.
 The SIP address must use the sip: prefix as well as a valid SIP domain; for example: `-SipAddress sip:kenmyer@litwareinc.com`.
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:301]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 7]
```diff
@@ -305,6 +317,9 @@ Accept wildcard characters: False
 ### -PassThru
 Enables you to pass a user object through the pipeline that represents the user whose account is being modified.
 By default, the `Set-CsUser` cmdlet does not pass objects through the pipeline.
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:321]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 8]
```diff
@@ -354,6 +369,8 @@ Accept wildcard characters: False
 ### -AcpInfo
 Enables you to assign one or more third-party audio conferencing providers to a user.
 However, it is recommended that you use the `Set-CsUserAcp` cmdlet to assign Audio conferencing providers.
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:373]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 9]
```diff
@@ -380,6 +397,8 @@ UseLyncArchivingPolicy
 ArchivingToExchange
 
 NoArchiving
+> [!WARNING]
+>This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:401]:
```suggestion
> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pbafna03, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

File: skype/skype-ps/skype/Set-CsUser.md [hunk 10]
```diff
@@ -124,7 +124,7 @@ If you later re-enable the account by using the Enabled parameter, the associate
 This differs from using the `Disable-CsUser` cmdlet to disable a user account.
 When you run the `Disable-CsUser` cmdlet, all the Skype for Business Server data associated with that account is deleted.
 > [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:126]:
```suggestion

```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 11]
```diff
@@ -124,7 +124,7 @@ If you later re-enable the account by using the Enabled parameter, the associate
...
->This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:127]:
```suggestion
**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 12]
```diff
@@ -227,7 +227,7 @@ The URI of the remote call control telephone gateway assigned to the user.
 The LineServerUri is the gateway URI, prefaced by "sip:".
 For example: sip:rccgateway@litwareinc.com
 > [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:229]:
```suggestion

```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 13]
```diff
@@ -227,7 +227,7 @@ The URI of the remote call control telephone gateway assigned to the user.
...
->This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:230]:
```suggestion
**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 14]
```diff
@@ -252,7 +252,7 @@ Private lines are often used for personal phone calls or for business calls that
 The private line value should be specified using the E.164 format, and be prefixed by the "TEL:" prefix.
 For example: TEL:+14255551297.
 > [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:254]:
```suggestion

```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 15]
```diff
@@ -252,7 +252,7 @@ Private lines are often used for personal phone calls or for business calls that
...
->This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:255]:
```suggestion
**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 16]
```diff
@@ -278,7 +278,7 @@ For details, see the parameter EnterpriseVoiceEnabled.
 
 To be enabled for remote call control, a user must have both a LineUri and a LineServerUri.
 > [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:280]:
```suggestion

```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 17]
```diff
@@ -278,7 +278,7 @@ For details, see the parameter EnterpriseVoiceEnabled.
...
->This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:281]:
```suggestion
**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 18]
```diff
@@ -298,7 +298,7 @@ Accept wildcard characters: False
 Unique identifier (similar to an email address) that allows the user to communicate using SIP devices such as Skype for Business.
 The SIP address must use the sip: prefix as well as a valid SIP domain; for example: `-SipAddress sip:kenmyer@litwareinc.com`.
 > [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:300]:
```suggestion

```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 19]
```diff
@@ -298,7 +298,7 @@ Accept wildcard characters: False
...
->This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:301]:
```suggestion
**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 20]
```diff
@@ -318,7 +318,7 @@ Accept wildcard characters: False
 Enables you to pass a user object through the pipeline that represents the user whose account is being modified.
 By default, the `Set-CsUser` cmdlet does not pass objects through the pipeline.
 > [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:320]:
```suggestion


```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 21]
```diff
@@ -318,7 +318,7 @@ Accept wildcard characters: False
...
->This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:321]:
```suggestion
**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 22]
```diff
@@ -370,7 +370,7 @@ Accept wildcard characters: False
 Enables you to assign one or more third-party audio conferencing providers to a user.
 However, it is recommended that you use the `Set-CsUserAcp` cmdlet to assign Audio conferencing providers.
 > [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:372]:
```suggestion

```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 23]
```diff
@@ -370,7 +370,7 @@ Accept wildcard characters: False
...
->This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:373]:
```suggestion
**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 24]
```diff
@@ -398,7 +398,7 @@ ArchivingToExchange
 
 NoArchiving
 > [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:400]:
```suggestion

```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 25]
```diff
@@ -398,7 +398,7 @@ ArchivingToExchange
...
->This parameter will not be available for Teams only tenants from version 3.0.0 onwards.
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:401]:
```suggestion
**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @pbafna03, some more, please.  Because PowerShell does not support markdown, [!NOTE], [!WARNING], etc. should be Note: <text>, Warning: <text>, etc. Also, suggest this should be a note, rather than a warning. Many thanks.

**Comment by pbafna03** [comment]
No worries, thanks @JohanFreelancer9

File: skype/skype-ps/skype/Set-CsUser.md [hunk 26]
```diff
@@ -98,6 +98,8 @@ Indicates whether the user is allowed to make audio/visual (A/V) calls by using
 If set to True, the user will largely be restricted to sending and receiving instant messages.
 
 You cannot disable A/V communications if a user is currently enabled for remote call control, Enterprise Voice, and/or Internet Protocol private branch exchange (IP-PBX) soft phone routing.
+> [!WARNING]
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:101]:
```suggestion

```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 27]
```diff
@@ -98,6 +98,8 @@ Indicates whether the user is allowed to make audio/visual (A/V) calls by using
...
+> This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:102]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 28]
```diff
@@ -222,6 +225,7 @@ Accept wildcard characters: False
 The URI of the remote call control telephone gateway assigned to the user.
 The LineServerUri is the gateway URI, prefaced by "sip:".
 For example: sip:rccgateway@litwareinc.com
+**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:228]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 29]
```diff
@@ -245,6 +249,7 @@ Private lines are often used for personal phone calls or for business calls that
 
 The private line value should be specified using the E.164 format, and be prefixed by the "TEL:" prefix.
 For example: TEL:+14255551297.
+**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:252]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 30]
```diff
@@ -269,6 +274,7 @@ To make and receive phone calls over the Internet, the user must be enabled for
 For details, see the parameter EnterpriseVoiceEnabled.
 
 To be enabled for remote call control, a user must have both a LineUri and a LineServerUri.
+**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:277]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 31]
```diff
@@ -287,6 +293,7 @@ Accept wildcard characters: False
 ### -SipAddress
 Unique identifier (similar to an email address) that allows the user to communicate using SIP devices such as Skype for Business.
 The SIP address must use the sip: prefix as well as a valid SIP domain; for example: `-SipAddress sip:kenmyer@litwareinc.com`.
+**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:296]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 32]
```diff
@@ -305,6 +312,8 @@ Accept wildcard characters: False
 ### -PassThru
 Enables you to pass a user object through the pipeline that represents the user whose account is being modified.
 By default, the `Set-CsUser` cmdlet does not pass objects through the pipeline.
+**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:315]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 33]
```diff
@@ -354,6 +363,7 @@ Accept wildcard characters: False
 ### -AcpInfo
 Enables you to assign one or more third-party audio conferencing providers to a user.
 However, it is recommended that you use the `Set-CsUserAcp` cmdlet to assign Audio conferencing providers.
+**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:366]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 34]
```diff
@@ -380,6 +390,7 @@ UseLyncArchivingPolicy
 ArchivingToExchange
 
 NoArchiving
+**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:393]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

File: skype/skype-ps/skype/Set-CsUser.md [hunk 35]
```diff
@@ -121,6 +123,7 @@ If you disable an account by using the Enabled parameter, the information associ
 If you later re-enable the account by using the Enabled parameter, the associated account information will be restored.
 This differs from using the `Disable-CsUser` cmdlet to disable a user account.
 When you run the `Disable-CsUser` cmdlet, all the Skype for Business Server data associated with that account is deleted.
+**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Set-CsUser.md:126]:
```suggestion

**Note**: This parameter is not available for Teams Only tenants from version 3.0.0 onwards.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Apologies again @pbafna03, a blank line is needed to prevent the note from becoming part of the preceding line. Thanks again.

**Review by JohanFreelancer9 — APPROVED** [review]
@pbafna03, thanks so much for committing the changes and your quick response. Appreciate it.

**Review by dariomws — CHANGES_REQUESTED** [review]
@pbafna03, I don't think this note is required because all these parameters are documented as not applicable to SfBO. They are in the code but when you try to use them you get:
image

Except for AudioVideoDisabled and PassThru, you can change them.

We should apply the following updates to this article:
- If the parameter is not available in Teams PS, document that it's not applicable to it is enough for me.
- If the parameter is available in Team PS, document that it's applicable to "Microsoft Teams" instead of SfBO is enough for me.

**Comment by dariomws** [comment]
@pbafna03, after this PR I have reviewed the following one:
https://github.com/MicrosoftDocs/office-docs-powershell/pull/8834

If Set-CsUser will be deprecated, I think that note is enough and we don't need to merge this one.

**Comment by pbafna03** [comment]
> @pbafna03, after this PR I have reviewed the following one: #8834
> 
> If Set-CsUser will be deprecated, I think that note is enough and we don't need to merge this one.

Hi @dariomws - the overall Set-CsUser deprecation will take a bit of time (targeted for completion in Apr'22), but the unsupported params are part of flight and have already started not working for Teams only Tenants. Hence thought it would be good to explicitly callout.

**Review by dariomws — APPROVED** [review]
@pbafna03, maybe you should use the same note:

**Note**: This cmdlet will be deprecated. Use the new [Set-CsPhoneNumberAssignment](/powershell/module/teams/set-csphonenumberassignment) cmdlet instead.

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**